### DISCUSSION TITLE
入绳网

### DISCUSSION THREAD

**Discussion by Minapocalis** [share121/inter-knot#4895] [委托]
这怎么设头像啊，有点搞不懂
![1724658633424](https://github.com/user-attachments/assets/65c384b3-30a6-4677-8187-b76c7aa7faf5)

**Comment by HLXS-HLXS** [comment]
GitHub官网左上角点头像，点设置，点profile picture【个人资料图片】下面的edit【编辑】，点upload a photo…【上传图片】，选择图片双击上传，图片不能大于1mb

**Comment by FeiXiao-zm** [comment]
先这样再那样就可以了
![Uploading 1726383364540.png…]()
